src/main/kotlin/se/uulm/snowballr/backend/repository/RepoResultHelper.kt

```diff
@@ -0,0 +1,44 @@
+package se.uulm.snowballr.backend.repository
+
+import se.uulm.snowballr.backend.model.EntityType
+import se.uulm.snowballr.backend.model.SnowballRException.NotFoundException
+import java.util.UUID
+
+/**
+ * Returns an entity by its ID encapsulated by a [Result]. If the entity couldn't be found a [Result.Failure] is
```

Comment:
```suggestion
 * Returns an entity by its ID, encapsulated by a [Result]. If the entity couldn't be found, a [Result.Failure] is
```
Or am I mistaken? (same would apply for second comment)

Reply:
The first comma is not correct, but the second one is.